### ISSUE TITLE
Sigma Parameter Behaves as Expected? (update documentation for groupFWHM?)

### ISSUE THREAD

**Issue by ColeWunderlich** [sneumann/CAMERA#13]
I am not sure the sigma parameter for `groupFWHM()` is behaving as expected.  In the man page it states that `sigma = the multiplier of the standard deviation`  However, in the code for `groupFWHM` sigma acts as a divisor, not a multiplier. 

Source Code:
`hwhm   <- ((rt.max-rt.min) / sigma * 2.35 * perfwhm) / 2; #fwhm of the highest peak`

Even if everything is working as intended, I would suggest changing the description to : `The FWHM (full width at half maximum) of a peak is estimated as FWHM = (rt.max-rt.min)/sigma * 2.35` in order to make things more clear.

Another option could be adding: ` The SD (standard deviation) is estimated as SD = (rt.max-rt.min)/sigma` instead of the suggestion above.

**Comment by sneumann** [comment]
This referring to the code at https://github.com/sneumann/CAMERA/blob/118bfb996f8f56bf10fc0bcf91d2d4a8930faa65/R/xsAnnotate.R#L305

**Comment by ColeWunderlich** [comment]
Agreed.  The man page has no mention of how sigma factors in to the FWHM calculation, so I think combining `sigma` and `perfwhm` into one parameter and updating the man page describing the new parameter is the best way to go.
